transports/webrtc/src/lib.rs

```diff
@@ -79,12 +79,7 @@
 //! hand-crate the SDP answer generated by the remote, this is problematic. A way to solve this
 //! is to make the hash a part of the remote's multiaddr. On the server side, we turn
 //! certificate verification off.
-
-mod proto {
-    #![allow(unreachable_pub)]
-    include!("generated/mod.rs");
-    pub(crate) use self::webrtc::pb::{mod_Message::Flag, Message};
-}
-
 #[cfg(feature = "tokio")]
 pub mod tokio;
+
+pub use libp2p_webrtc_utils as utils;
```

Comment:
The utils crate should _not_ be in the public API.

Reply:
Removed